### PR TITLE
increased brokerKpiEvent size to 20MB

### PR DESCRIPTION
**Pull request by Vadym-Lopatka**

### CODE REVIEW

**Comment by selina-bot-user** [comment]
:star: PR built and available in a preview **selina-finance-ms-lending-api-pr-83** [here](http://ms-lending-api-pr83.selinadev.co.uk)

File: src/main/java/com/selina/lending/config/KafkaProducerConfig.java [hunk 1]
```diff
@@ -48,6 +50,7 @@ public ProducerFactory<String, Object> multiTypeProducerFactory() {
         configProps.put(BOOTSTRAP_SERVERS_CONFIG, bootstrapServers);
         configProps.put(KEY_SERIALIZER_CLASS_CONFIG, StringSerializer.class);
         configProps.put(VALUE_SERIALIZER_CLASS_CONFIG, JsonSerializer.class);
+        configProps.put(MAX_REQUEST_SIZE_CONFIG, MAX_MESSAGE_SIZE);
```
Comment (bpoon-selina) [inline src/main/java/com/selina/lending/config/KafkaProducerConfig.java:53]: What happens to the existing topic? Does it automatically size to 20mb with no negative impact?

**Comment by selina-bot-user** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[bpoon-selina](https://github.com/Selina-Finance/ms-lending-api/pull/83#issuecomment-1309971353 "Approved")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=Selina-Finance%2Fms-lending-api).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/Selina-Finance/ms-lending-api/blob/main/OWNERS)~~ [bpoon-selina]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**